main/lsp/lsp_helpers.cc

```diff
@@ -90,18 +90,14 @@ string LSPLoop::fileRef2Uri(const core::GlobalState &gs, core::FileRef file) {
 } // namespace sorbet::realmain::lsp
 
 unique_ptr<Range> loc2Range(const core::GlobalState &gs, core::Loc loc) {
```

Comment:
Can you add a comment in `lsp.h` above the definitions of `loc2Range` and `loc2Location` that they return `nullptr` if the `loc` doesn't exist?